.travis.yml

```diff
@@ -15,9 +15,19 @@ before_install:
   - curl $VERTICA_CE_URL --create-dirs -o docker-vertica/packages/vertica-ce.latest.rpm
   - docker build -f docker-vertica/Dockerfile.centos.7_9.x --build-arg VERTICA_PACKAGE=vertica-ce.latest.rpm -t jbfavre/vertica docker-vertica
   - docker images
-  - docker run -d -p 5433:5433 jbfavre/vertica
+  - docker run --hostname vertica-docker --name vertica_docker -d -p 5433:5433 jbfavre/vertica
   - sleep 60
 
 script:
   - go test --locator localhost:5433 --user dbadmin -race .
+  - go test --locator localhost:5433 --user dbadmin --use_prepared_statements=0 -race .
   - go test -race ./logger
```

Comment:
This is the place to trigger test suites. Is it better to move the following to a test suite (e.g.  a collection of connection tests)?

Reply:
I'm not sure. This is easier for me to do it like this to try every combination of flags and then try every combination again once we've rebooted the server in SSL mode.

If there's an easier way of doing this, can you merge and change it?